src/App.jsx

```diff
@@ -91,7 +91,7 @@ function App() {
   const isAppLoading = useSelector(state => state.app.loading);
   const isAppLoaded = useSelector(state => typeof state.app.marketPrice != "undefined"); // Hacky way of determining if we were able to load app Details.
   const { bonds } = useBonds();
-  async function loadDetails(whichDetails) {
+  async function loadDetails(whichDetails: string) {
```

Comment:
probs don't need this because its a .jsx

Reply:
My vscode always yelling at me that it needs type declared. That's why I changed it: 
![image](https://user-images.githubusercontent.com/80423742/134185109-db56be9d-ba49-46e1-b3d5-8435a8d78a37.png)